### PR TITLE
Fixes #22404: Rhel7 builds fails due to a file with spaces in its name

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/22404

### CODE REVIEW

File: rudder-api-client/SPECS/rudder-api-client.spec [hunk 1]
```diff
@@ -66,17 +66,6 @@ Requires: python3, python3-requests, python3-docopt, python3-urllib3
 %description
 Command line tools and python libraries to call Rudder.
 
-#=================================================
-# Source preparation
-#=================================================
-%prep
-%setup -c
```
Comment (amousset) [inline rudder-api-client/SPECS/rudder-api-client.spec:73]: this line does something

File: rudder-relay/SPECS/rudder-relay.spec [hunk 2]
```diff
@@ -118,20 +118,6 @@ Rudder is an open source configuration management and audit solution.
...
 run a Rudder relay server on a machine.
 
-#=================================================
-# Source preparation
-#=================================================
-%prep
-%setup -c
-
-# We don't know the exact version
-cd rudder-sources-*/rudder/relay/sources/
```
Comment (amousset) [inline rudder-relay/SPECS/rudder-relay.spec:128]: this is required

Comment (VinceMacBuche) [reply rudder-relay/SPECS/rudder-relay.spec:128]: what ? setup-c is required, but not the cd

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
I added again the setup -c lines

File: rudder-relay/SPECS/rudder-relay.spec (hunk 2 above)
Comment (amousset) [reply rudder-relay/SPECS/rudder-relay.spec:128]: indeed

**Comment by VinceMacBuche** [comment]
OK, merging this PR

**Pull request merged**